operator/src/main/java/oracle/kubernetes/weblogic/domain/model/DomainCondition.java

```diff
@@ -55,6 +56,9 @@
   @NotNull
   private String status = "True";
 
+  // internal: used to select failure conditions for deletion
+  private volatile boolean marked;
```

Comment:
The comment helps, but the field name could be more informative, IMO.

Reply:
will change